integration-cli/docker_api_exec_test.go

```diff
@@ -100,12 +99,12 @@ func (s *DockerSuite) TestExecAPIStartBackwardsCompatible(c *check.C) {
 }
 
 // #19362
-func (s *DockerSuite) TestExecAPIStartMultipleTimesError(c *check.C) {
-	dockerCmd(c, "run", "-d", "--name", "test", "busybox", "top")
+func (s *DockerSuite) TestExecApiStartMultipleTimesError(c *check.C) {
+	runSleepingContainer(c, "-d", "--name", "test")
 	execID := createExec(c, "test")
 	startExec(c, execID, http.StatusOK)
 
-	timeout := time.After(10 * time.Second)
+	timeout := time.After(60 * time.Second)
```

Comment:
Is ten seconds cutting it too close on windows? Seemed fine in your example output.